fbw-a32nx/src/systems/instruments/src/MCDU/legacy/A32NX_FMCMainDisplay.ts

```diff
@@ -318,6 +320,8 @@ export abstract class FMCMainDisplay implements FmsDataInterface, FmsDisplayInte
   public casToMachManualCrossoverCurve;
   public machToCasManualCrossoverCurve;
 
+  private readonly lateralMode = RegisteredSimVar.create('L:A32NX_FMA_LATERAL_MODE', SimVarValueType.Number);
```

Comment:
We should really use `A32NX_FMGC_{number}_DISCRETE_WORD_2` instead of the legacy var.